client/components/tinymce/plugins/insert-menu/add-instagram-shortcode.jsx

```diff
@@ -0,0 +1,12 @@
+import menuItemPlugin from './menu-item-plugin';
+
+import InstagramShortcodeWizard from 'post-editor/wizards/instagram';
+
+export const InstagramShortCodePlugin = menuItemPlugin( {
+	Wizard: InstagramShortcodeWizard,
```

Comment:
thanks @gwwar - I was following React's naming convention here since `Wizard` is a React component